db/migrate/20160901010430_change_column_type.rb

```diff
@@ -1,13 +0,0 @@
-class ChangeColumnType < ActiveRecord::Migration
```

Comment:
I think this removes some of @mailethiesen's work.  I think we need this file in here.